exporter/management/commands/wiper.py

```diff
@@ -8,7 +8,12 @@
 
 
 class Command(BaseCommand):
-    help = 'Removes all the exported data'
+    """
+    Starts wiper worker (connects to rabbitmq and consumes messages) responsible for removal
+    of previously exported data. Removes data of particular job (defined in message) only.
```

Comment:
I don't understand. Where is the job to wipe indicated?

Reply:
This is command to start worker. "Starts wiper worker..." The job is defined in the message "job (defined in message)"